src/test/java/cruise/umple/sample/downloader/ConsoleMainTest.java

```diff
@@ -17,19 +14,16 @@
 import com.google.inject.Inject;
 import com.google.inject.Provider;
 
-import cruise.umple.sample.downloader.ConsoleMain.ImportRuntimeData;
+import cruise.umple.sample.downloader.consistent.ImportRepository;
+import cruise.umple.sample.downloader.consistent.ImportRepositorySet;
 import cruise.umple.sample.downloader.util.MockDocumentFactoryModule;
-import cruise.umple.sample.downloader.util.Pair;
 
 /**
  * Created by kevin on 15-02-23.
  */
-@Guice(modules={DownloaderModule.class, MockDocumentFactoryModule.class})
+@Guice(modules={MockDocumentFactoryModule.class})
```

Comment:
Forgot to note, the C++ version of attributes are extremely primitive and are only for the compiler as no runtime reflection engine exists.